scripts/tools/zap/zap_execution.py

```diff
@@ -25,6 +25,15 @@
 #
 MIN_ZAP_VERSION = '2023.4.27'
 
+ZAP_INSTALL_BANNER = '*'*80 + """
+* You may need to install zap from https://github.com/project-chip/zap/releases
+* Please ensure one of these applies (see docs/guides/BUILDING.md for details):
+* - `zap-cli` is in your Python venv or $PATH.
```

Comment:
I would split venv and $PATH as they are sparate logic it seems.

We would also need to update `docs/code_generation.md` for this behavior change.

Reply:
Addressed via the other thread.